templates/eap-xp1-openjdk11-image-stream.json

```diff
@@ -0,0 +1,135 @@
+                        "annotations": {
+                            "description": "JBoss EAP XP Builder Image",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap,javaee,java,jboss,hidden",
+                            "supports": "eapxp:1.0,javaee:8,java:11",
+                            "sampleRepo": "https://github.com/jbossas/eap-quickstarts/openshift",
+                            "sampleContextDir": "kitchensink",
+                            "sampleRef": "7.3.x-openshift",
+                            "version": "1.0",
+                            "openshift.io/display-name": "JBoss EAP XP Builder Image with OpenJDK 11"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/jboss-eap-7/eap-xp1-openjdk11-openshift-rhel8:latest"
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-eap-xp1-openjdk11-runtime-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "JBoss EAP XP Runtime Image with OpenJDK 11",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "version": "1.0"
+                }
+            },
+            "labels": {
+                "xpaas": "1.0"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "latest",
+                        "annotations": {
+                            "description": "The latest released JBoss EAP XP runtime image.",
+                            "iconClass": "icon-eap",
+                            "tags": "runtime,eapxp,javaee,java,jboss,hidden",
+                            "supports": "eapxp:1.0,javaee:8,java:11",
+                            "sampleRepo": "https://github.com/jbossas/eap-quickstarts/openshift",
+                            "sampleContextDir": "kitchensink",
+                            "sampleRef": "7.3.x-openshift",
+                            "version": "latest",
+                            "openshift.io/display-name": "JBoss EAP XP Runtime Image with OpenJDK 11"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/jboss-eap-7/eap-xp1-openjdk11-runtime-openshift-rhel8:latest"
+                        }
+                    },
+                    {
+                        "name": "1.0",
```

Comment:
should we create a `1.0.0` version too? For EAP 7.3, we have `7.3` and `7.3.0`

Reply:
As mentioned above, this makes more sense for EAP i think, since we could have :7.3 meaning the latest release in the 7.3 stream, and :7.3.1 meaning the specific release. Here I think the most specific we'll ever have is :1.0, right?

Reply:
Well, :1.0-X with -X being the image build increment.